dotnet/Razorvine.Pyrolite/Pyrolite/Pickle/PickleUtils.cs

```diff
@@ -262,11 +262,7 @@ public static int bytes_to_integer(byte[] bytes, int offset, int size)
 			}
 			data=larger;
 		}
-		if(!BitConverter.IsLittleEndian) {
-			// reverse the byte array because pickle stores it little-endian	
-			Array.Reverse(data);
-		}
-		return BitConverter.ToInt64(data,0);
```

Comment:
FWIW, I hadn't changed these because they don't really show up in spark.  The types used there are all the binary versions of these.

Reply:
this one did not show for me as well, but when I changed everything to use Span instead of Array this change became mandatory to avoid `.ToArray` allocations